bin/wsweights/src/main.rs

```diff
@@ -338,7 +338,10 @@ async fn note_new_block(
     timestamp_cache: TimestampCache,
 ) -> Result<(), Box<dyn std::error::Error>> {
     let block_number = block.header().number;
-    let extrinsics_num = block.extrinsics().await?.len();
+    let extrinsics_num = block.extrinsics().await?.iter().map(|r| r.map(|tx| match tx.pallet_index() {
```

Comment:
the `.iter()` call might be heavy as it's doing some scale decoding and allocation
not sure about perf impact, but I don't expect it to be very high

Reply:
Will this still be able to capture "normal" transactions as well?